src/modules/activityLeaderboard/module.js

```diff
@@ -0,0 +1,77 @@
+import Factions from '../../constants/Factions';
+import { editMessage, factionDb, getEmoji, makeTitle, userDb } from '../../util';
+
+const enabled = true;
+
+const subscriptions = new Map();
+
+const updateActivity = () => {
+    const now = Date.now();
+    const users = userDb
+        .filter((user) => user.recentReset > now && user.activityRecent > 100)
+        .map((user, id) => [user, id])
+        .sort((a, b) => b[0].activityRecent - a[0].activityRecent);
+    const content =
+        users
+            .map(
+                ([user, id], i) =>
+                    `${makeTitle((i + 1).toString())} <@${id}> - ${user.activityRecent}`
+            )
+            .slice(0, 7)
```

Comment:
Wouldn't a slice before map be more efficient as this way no extra processing needs to be done for useless objects.

Reply:
Probbaly

Reply:
```suggestion
            .slice(0, 7)
            .map(
                ([user, id], i) =>
                    `${makeTitle((i + 1).toString())} <@${id}> - ${user.activityRecent}`
            )
```